refactor_persis_layer.html

```
@@ -924,26 +926,30 @@ <h2 class="text-2xl font-bold mb-2">Collection Naming &amp; ID Conventions</h2>
-  E["Step 5\nPort dagrun store\n(most complex)"] -->
-  F["Step 6\nRemove file*/ packages\nSimplify context.go"] -->
-  G["Step 7\nAdd MemoryBackend\nin testutil/\nfor unit tests"]
-
-  style A fill:#f0f9ff,stroke:#7dd3fc
-  style B fill:#ecfdf5,stroke:#6ee7b7
-  style C fill:#ecfdf5,stroke:#6ee7b7
-  style D fill:#ecfdf5,stroke:#6ee7b7
-  style E fill:#ecfdf5,stroke:#6ee7b7
-  style F fill:#ecfdf5,stroke:#6ee7b7
-  style G fill:#faf5ff,stroke:#c4b5fd
+  A["✅ Step 1\nDefine interfaces\npersis/backend.go\npersis/codec.go\npersis/errors.go"] -->
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)